include/openenclave/edger8r/enclave.h

```diff
@@ -34,6 +34,13 @@ typedef void (*oe_ecall_func_t)(
     size_t output_buffer_size,
     size_t* output_bytes_written);
 
+/**
+ * Get the internal status of the enclave.
+ *
+ * @returns read-only copy of the internal status.
+ */
+oe_result_t oe_get_enclave_status();
```

Comment:
This looks good. Can you make this a no-op for non-sgx similar to oe_lfence?

Reply:
Ah! Good catch.